react/Banner.js

```diff
@@ -14,22 +27,30 @@ class Banner extends Component {
       page,
       params,
       mobileHeight,
+      typeOfRoute,
     } = this.props
 
+    const content = (
+      <div className="img-container">
+        <img className="img-regular w-100" src={image} alt={description} />
+        <div
+          className="img-mobile"
+          style={{ maxHeight: `${mobileHeight}px` }}
+        >
+          <img className="w-100" src={mobileImage} alt={description} />
+        </div>
+      </div>
+    )
+
     return (
-      <div>
-        <Link page={page} params={params}>
-          <div className="img-container">
-            <img className="img-regular w-100" src={image} alt={description} />
-            <div
-              className="img-mobile"
-              style={{ maxHeight: `${mobileHeight}px` }}
-            >
-              <img className="w-100" src={mobileImage} alt={description} />
-            </div>
-          </div>
+      typeOfRoute === 'internal' ?
+        <Link page={page} params={this.getParams(params)}>
+          {content}
         </Link>
-      </div>
+        :
+        <a href={page} target="_blank">
```

Comment:
You encapsulated the content div inside the anchor tag, destructuring the CSS styling.